src/ServiceControl.Monitoring/Timings/LongValueOccurrenceHandler.cs

```diff
@@ -23,16 +23,16 @@ public Task Handle(LongValueOccurrences message, IMessageHandlerContext context)
         {
             try
             {
-                var endpointInstanceId = EndpointInstanceId.From(context.MessageHeaders);
+                var instanceId = EndpointInstanceId.From(context.MessageHeaders);
```

Comment:
Would it make sense to collapse this with `TaggedLongValueOccurrenceHandler` into a single `TimingsHandler`?